xdem/coreg/workflows.py

```diff
@@ -173,7 +175,7 @@ def dem_coregistration(
     :param src_dem_path: Path to the input DEM to be coregistered
     :param ref_dem_path: Path to the reference DEM
     :param out_dem_path: Path where to save the coregistered DEM. If set to None (default), will not save to file.
-    :param coreg_method: Coregistration method or pipeline.
+    :param coreg_method: The xdem coregistration method, or pipeline.
```

Comment:
I think we can state simply "Coregistration method" here, it's implicit we're working with xDEM :wink: